internal/dispatch/keys/hasher_common.go

```diff
@@ -27,9 +27,7 @@ type hashableRelationReference struct {
 }
 
 func (hrr hashableRelationReference) AppendToHash(hasher hasherInterface) {
-	hasher.WriteString(hrr.Namespace)
-	hasher.WriteString("#")
-	hasher.WriteString(hrr.Relation)
+	hasher.WriteString(hrr.Namespace + "#" + hrr.Relation)
```

Comment:
are we sure this better than multiple WriteString calls?

Reply:
Yep!